pkg/scheduler/cache/ports.go

```diff
@@ -0,0 +1,132 @@
+/*
```

Comment:
Isn't it better to call this file host_ports.go to distinguish it from container ports?

Reply:
But there also has `ProtocolPort` code, so ports.go maybe more accurate.

Reply:
+1 to host_ports, we only care about host port in scheduler.